src/app/containers/PageHandlers/withLoading.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { bool } from 'prop-types';
+import {
+  GhostWrapper,
+  GridItemConstrainedMedium,
+  Container,
+} from '../../lib/styledGrid';
+
+const WithLoading = Component => {
+  return function WithLoadingComponent({ loading, ...props }) {
+    if (!loading) return <Component {...props} />;
+    return (
+      <main role="main">
```

Comment:
This ideally would be a separate component that defines the loading page layout. This would be re-factored when we get some official UX input